dask/array/core.py

```diff
@@ -937,6 +941,9 @@ def store(sources, targets, lock=True, regions=None, compute=True,
 
         load_dsks_mrg = sharedict.merge(store_dsks_mrg, *load_dsks)
 
+        from ..optimization import cull, inline
```

Comment:
Trying to import these outside the function results in an `ImportError` during testing. So imported them locally only where they are needed to avoid this issue.

Reply:
Do we know what the import issue is?  I would expect the `optimization` module to be pretty well isolated.

Reply:
Not really sure. Initially this issue forced me to keep it in the function. Now I'm having trouble reproducing it. 😕

Reply:
Have moved the `import` out now and it seems to work fine. Not sure what was going on during development.